tools/cockpit.spec

```diff
@@ -509,6 +509,13 @@ restorecon /usr/libexec/cockpit-wsinstance-factory
 %postun ws
 %systemd_postun_with_restart cockpit.socket
 %systemd_postun_with_restart cockpit.service
+if [ "$1" = "0" ]; then
+  # When $1 is 0, this is an uninstallation.
+  getent passwd cockpit-ws >/dev/null && userdel cockpit-ws
+  getent group cockpit-ws >/dev/null && groupdel cockpit-ws
```

Comment:
That is unsafe, and must not be done. Removing system users is always very delicate, and it's only safe if there are absolutely *no* files owned by these in the file system. In this case there are -- `/etc/cockpit/ws-certs.d/`. Removing the system user means the next time a system group is created, it has access to cockpit's private keys.